tests/write_file/BUILD

```diff
@@ -0,0 +1,83 @@
+load("//rules:write_file.bzl", "write_file")
+
+package(default_testonly = 1)
+
+sh_test(
+    name = "write_file_tests",
+    srcs = ["write_file_tests.sh"],
+    data = [
+        ":run_executables",
+        # Use DefaultInfo.files from 'write_empty_text' (via 'file_deps').
+        ":file_deps",
```

Comment:
There are not alphabetical, which is a sign that you haven't yet run buildifier. :) Be sure to do so (and retain the comment ordering)

Reply:
I have. Buildifier doesn't reorder these lines.